dev-libs/hyprgraphics/hyprgraphics-0.1.3.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 2023-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Hyprland graphics / resource utilities"
+HOMEPAGE="https://github.com/hyprwm/hyprgraphics"
+SRC_URI="https://github.com/hyprwm/${PN}/archive/v${PV}.tar.gz -> ${P}.gh.tar.gz"
+
+LICENSE="GPL-3"
```

Comment:
Taking a look at the repo, it's not GPL-3.

Reply:
Thanks, I have updated the stable ebuild as well.